### PR TITLE
Validated data

### PR DESCRIPTION
**Pull request by gidsi**
Adds `validatedJson`

### CODE REVIEW

File: v2/validator.go [hunk 1]
```diff
@@ -33,6 +33,7 @@ type urlValidationResponse struct {
 	Cors         bool          `json:"cors"`
 	ContentType  bool          `json:"contentType"`
 	CertValid    bool          `json:"certValid"`
+	ValidatedJson interface{} `json:"validatedJson,omitempty"`
```
Comment (dbrgn) [inline v2/validator.go]: Your horizontal alignment seems to be off 🙂

File: v2/validator.go [hunk 2]
```diff
@@ -44,6 +45,7 @@ type schemaError struct {
 type jsonValidationResponse struct {
 	Valid        bool          `json:"valid"`
 	Message      string        `json:"message"`
+	ValidatedJson interface{} `json:"validatedJson,omitempty"`
```
Comment (dbrgn) [inline v2/validator.go]: Here too.

**Review by dbrgn — APPROVED** [review]
Only some nitpicking. Good otherwise.

File: v2/validator.go (hunk 1 above)
Comment (gidsi) [reply v2/validator.go]: wops, forgot to run go fmt!
One day i will add it to my IDE, really! :joy:

**Pull request merged**